src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Third-Party/Gutenberg" />
+
+# WordPress Gutenberg Blocks
+
+The Wordpress Gutenberg editor is a block-based page builder. The editor comes with
```

Comment:
Can we link to some documentation somewhere in here?

Reply:
Added a link to the main page on wordpress.org, but if you'd like to link to something more specific to development, let me know.